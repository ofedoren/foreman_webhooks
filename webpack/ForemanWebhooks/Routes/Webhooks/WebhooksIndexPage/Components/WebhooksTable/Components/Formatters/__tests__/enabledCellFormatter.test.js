import { enabledCellFormatter } from '../index';

describe('enabledCellFormatter', () => {
  it('render', () => {
    expect(enabledCellFormatter()(true)).toMatchSnapshot();
  });
});
