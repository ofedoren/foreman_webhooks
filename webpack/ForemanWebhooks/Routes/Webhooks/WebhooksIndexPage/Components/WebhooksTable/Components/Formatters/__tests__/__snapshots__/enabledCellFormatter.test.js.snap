// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enabledCellFormatter render 1`] = `
<EnabledCell
  condition={true}
/>
`;
